docs/install-macos.md

```diff
@@ -20,6 +20,12 @@ Once homebrew is installed use it to install [pipx](https://pipxproject.github.i
 brew install pipx pyenv
 ```
 
+Add pipx to your path with:
+
+```bash
+echo 'export PATH="$HOME/.local/bin/:$PATH"' >> .bashrc
```

Comment:
would you need to restart your shell session for this to take effect?

Reply:
ah, of course, we'll need to source bashrc for it take effect, thanks!

Reply:
Fixed and rebased, also caught that I wasn't explicitly setting the path to `.bashrc`.

Reply:
much better, thanks!